src/components/main-form.js

```diff
@@ -50,6 +50,35 @@ option2.term = 30
 
 // CSS
 
+const Summary = styled(Row)`
+  b {
+    font-weight: 500;
+  }
+
+  p,
+  h5 {
+    font-size: 17px;
+  }
+
+  h5 {
+    margin-bottom: 1rem;
+    margin-left: 1.5rem;
+  }
+
+  .col {
+    margin-left: -2rem;
+  }
+
+  ol {
+    font-size: 15px;
```

Comment:
q: What do things look like w/o any custom font-size changes?

Reply:
relatively disproportionate in my opinion. You can give it a try.

Reply:
After removing some custom font sizes, I think it looks still fine:

![image](https://user-images.githubusercontent.com/3756871/64924603-666f8c80-d79b-11e9-8bfb-a2d7a1bd1c9b.png)

Another thing I've seen people do is to reduce the blackness of the body font to reduce emphasis. Perhaps we can give that a try too?